### DISCUSSION TITLE
1000-em vs 2048-em symbols

### DISCUSSION THREAD

**Discussion by jirutka** [ryanoasis/nerd-fonts#872] [Q&A]
What does this really mean and what implications does it have? If I use Symbols Nerd Font with fontconfig on Linux, how do I decide which variant (1000-em vs 2048-em) is the best? What if both variants are installed, is fontconfig able to select the right variant for particular font automatically?

**Comment by bitterhalt** [comment]
+1. I would like to know this too!

**Comment by lack** [comment]
I'm not sure about this, but it seems like the 1000-em versus 2048-em difference is more about the resolution of the individual glyphs in the font.

The 2048-em is like a higher-resolution version of a photo.

At small sizes they're indistinguishable, but if you're using a huge font size the 2048-em version will look better. Probably...

**Reply by Finii** [reply]
That is indeed the case, at least were the glyph source has a higher resolution.

But still (for me) the question, why not drop the 1000 EM version?
The size difference is negligible
```
-rw-rw-r-- 1 fini fini 869296 Jan 13 11:59 'Symbols-1000-em Nerd Font Complete.ttf'
-rw-rw-r-- 1 fini fini 920548 Jan 13 11:59 'Symbols-2048-em Nerd Font Complete.ttf'
```

But then there is this comment...
>    If you feel you need more accuracy in TrueType you can change the em-size to   8096.
>
>    You can do this on PostScript Type1 fonts too, however, there are some
   applications which assume that all OpenType postscript fonts have an em-size
   of 1000, so it is best not to do this for OpenType output.

and this warning in `fontforge`

```
fontforgeexe/savefontdlg.c:         if ( gwwv_ask(_("Non-standard Em-Size"),(const char **) buts,0,1,_(
"The convention is that PostScript fonts should have an Em-Size of 1000.
But this font has a size of %d. This is not an error, but you might consider
altering the Em-Size with the Element->Font Info->General dialog.\n
Do you wish to continue to generate your font in spite of this?"),
```

Which might or might not be the reason we have the 1000 EM font.

I _believe_ this does not affect us in any way, and we could safely drop the 1000 EM versions.
I never bothered with them.
